cdhweb/projects/models.py

```diff
@@ -279,6 +281,14 @@ def get_sitemap_urls(self, request):
             urls[0]["priority"] = 0.6
         return urls
 
+    def display_tags(self):
```

Comment:
@haydngreatnews I know that this is extra, but could you please document that a project's tags are meant to be a collection of its methods, fields and roles? Since the property names are quite generic, it's difficult to read at a glance. A unit test for this function would also be appreciated.